_includes/_blogcontent/2024-02-26-userfeedback-2024.md

```diff
@@ -0,0 +1,68 @@
+
+The bullet points below encapsulate direct responses to the inquiries raised by the community during the ADS team’s conference interactions.
+
+* **ADS Is NOT Going Away:** While transitioning to NASA SciX, ADS will remain accessible in [its current format](https://ui.adsabs.harvard.edu/), preserving familiarity and ensuring uninterrupted access for users.
+* **ADS Support Will Continue:** The ADS team reaffirms its commitment to user assistance and assures users of continued support during the transition. We will be a reliable resource for both existing ADS users and new NASA SciX users.
+* **Astrophysics Remains a Core Focus:** The astrophysics focus within NASA SciX aligns with ADS's original purpose. Services initially designed for astrophysics are, and will continue to be, a firm foundation for the development of similar services for other scientific disciplines.
+* **Innovative & Intuitive SciX Search:** The user-friendly, discipline-specific search features planned for NASA SciX aim to enhance the search experience, catering to users' specific interests with precision.
+* **Interdisciplinary Insights:** NASA’s charge that SciX facilitate interdisciplinary exploration matches with user aspirations, ensuring a seamless integration of relevant data from various scientific fields.
+* **Informatics Innovation:** NASA SciX inherits from ADS the aim of innovation in informatics innovation, which will ensure that NASA SciX evolves dynamically, introducing new features and tools to enhance the overall user experience.
+* **Expanded Staff Expertise:** The ADS team acknowledges the need for added expertise as we transition into NASA SciX, because we remain dedicated to supporting users effectively.
+* **User Feedback:** The ongoing [call for user feedback](https://scixplorer.org/feedback/general?from=%2F) ensures that NASA SciX remains attuned to the diverse needs of the scientific community, including astronomers.
+
+### Conclusion: A Collaborative Transition
+In conclusion, the shift from ADS to NASA SciX is a collaborative effort between the ADS team and our user community. Shaped by survey insights, ADSUG discussions, conference engagements, and your comments, the NASA SciX team’s user-centric development works to ensure it meets community needs. NASA SciX aims to preserve ADS strengths while pioneering new avenues for science discovery. As the ADS/SciX team looks ahead, your input remains key in shaping the future of NASA SciX. Share your thoughts, insights, and 
```

Comment:
Whoops look like the last line got cut off here

Reply:
added missing text